Frameworks/AudioToolbox/SystemSound.mm

```diff
@@ -16,62 +17,273 @@
 
 #import <AudioToolbox/SystemSound.h>
 #import <StubReturn.h>
+#import <Starboard.h>
+#import <UWP/WindowsMediaPlayback.h>
+#import <mutex>
+
+
+// Calls into C++ to access Windows.Phone.Devices.Notification namespace to vibrate devices with vibration capability.
+void vibrate();
+
+static const wchar_t* TAG = L"AudioToolbox";
+
+// We generate SystemSoundIDs starting with 5001 as some system supplied sounds have ids below 5000.
+static const SystemSoundID c_userSSID = 5000;
```

Comment:
As a more concrete example, a casual observer may see SSID and incorrectly it refers to Service Set Identifier (from WIFI) and be really really confused what this code is doing.